src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/RelationRewriteRule.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bag2deposit.ddm
+
+import better.files.File
+import nl.knaw.dans.easy.bag2deposit.parseCsv
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import org.apache.commons.csv.CSVFormat
+
+import scala.xml.{ Attribute, Elem, Node, Null }
+import scala.xml.transform.RewriteRule
+
+case class RelationRewriteRule(cfgDir: File) extends RewriteRule with DebugEnhancedLogging {
+
+  val easyRef = "https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:"
+  val relations = List(
+    "relation",
+    "conformsTo",
+    "hasFormat",
+    "hasPart",
+    "hasVersion",
+    "isFormatOf",
+    "isPartOf",
+    "isReferencedBy",
+    "isReplacedBy",
+    "isRequiredBy",
+    "isVersionOf",
+    "references",
+    "replaces",
+    "requires"
+  )
+
+  def isEasyDatasetReference(node: Node) = {
+    val attr = node.attributes
+    node.prefix ==  "ddm" && relations.contains(node.label) &&
```

Comment:
You're right. it could theoretically also be a dc or dcterms with just a text. i've only looked in the href, because i saw in the list of relations that it is always placed in the href attribute